apps/dataflow/src/scraper/fms/download_svg_from_s3.py

```
@@ -0,0 +1,48 @@
+
+import os
+from s3_utils import client, bucket_name
+
+# S3 source path
+S3_SOURCE = "floorplan_svg/"
+# Local destination folder
+LOCAL_DESTINATION = "floorplan_svg_downloaded"
+
+
+def download_all_svg_files():
+    """Download all SVG files from S3 floorplan_svg folder"""
```

Ruff (0.14.14)

[warning] First line should end with a period

Add period

(D400)